### PR TITLE
Add health probe for Karmada-agent

### PR DESCRIPTION
**Pull request by Poor12**
Signed-off-by: Poor12 <shentiecheng@huawei.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Add health probe for Karmada-agent

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
None

### CODE REVIEW

**Comment by lonelyCZ** [comment]
Thanks @Poor12 , I will review it ASAP.

/assgin

**Comment by prodanlabs** [comment]
hi @Poor12 ，Can you update the `init` example, thanks.
https://github.com/karmada-io/karmada/blob/master/pkg/karmadactl/cmdinit/utils/examples.go#L12

**Comment by Poor12** [comment]
> hi @Poor12 ，Can you update the `init` example, thanks. https://github.com/karmada-io/karmada/blob/master/pkg/karmadactl/cmdinit/utils/examples.go#L12

Done.

**Comment by prodanlabs** [comment]
> Done.

Thanks @Poor12

File: pkg/karmadactl/cmdinit/utils/examples.go [hunk 1]
```diff
@@ -76,7 +76,18 @@ spec:
             - --karmada-context=%s
             - --cluster-name={member_cluster_name}
             - --cluster-status-update-frequency=10s
+			- --bind-address=0.0.0.0
```
Comment (lonelyCZ) [inline pkg/karmadactl/cmdinit/utils/examples.go:79]: This is not aligned. Others are fine.

**Comment by lonelyCZ** [comment]
Thanks @Poor12 

/lgtm

**Pull request merged**